.github/CODEOWNERS

```
@@ -9,4 +9,4 @@
 #
 #
 # Global Owners
-* @flands @MrAlias
+* @signalfx/gdi-specification-approvers @signalfx/gdi-specification-maintainers
```

Shouldn't it be

```suggestion
* @signalfx/gdi-specification-approvers @signalfx/gdi-specification-maintainers

.github/CODEOWNERS @signalfx/gdi-specification-maintainers
```

to match https://github.com/signalfx/gdi-specification/pull/88?

---

```suggestion
* @signalfx/gdi-specification-approvers @signalfx/gdi-specification-maintainers

CODEOWNERS @signalfx/gdi-specification-maintainers
```